lib/yard/docstring.rb

```diff
@@ -102,6 +106,7 @@ def self.new!(text, tags = [], object = nil, raw_data = nil, ref_object = nil)
     #   with.
     def initialize(content = '', object = nil)
       @object = object
+      @namespace = object
```

Comment:
It _kind of_ seems like `.namespace` is just a dupe of `.object`? Or, at the very least, that namespace could be a combination of `@object || @unresolved_reference` and thus no attribute would be needed.